product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -150,19 +154,23 @@ The URL contains the following information:
 
 -   `postgres_type` — The subprotocol is the Postgres type. Specify `edb` if you're using Advanced Server or `postgresql` if you're using PostgreSQL. 
 
--   `<host_name>` — The host name of your cluster. You can copy it from the **Host** field on the **Connect** tab in the BigAnimal portal.
+-   `<host_name>` — The host name of your cluster. You can copy it from the **Host** field on the **Connect** tab in the Cloud Service portal.
+
+-   `<port>` — The port number that the database listener is monitoring. You can copy it from the **Port** field on the **Connect** tab in the Cloud Service portal.
+
+-   `<database_id>` — The name of the target database.  Set this to the name of the database in your cluster that you want to use as your migration target database.  The name of the default database for your cluster is shown in the **Dbname** field on the **Connect** tab in the Cloud Service portal.  Often a separate database is created for use as the migration target.
 
--   `<port>` — The port number that the database listener is monitoring. You can copy it from the **Port** field on the **Connect** tab in the BigAnimal portal.
+Ensure these values are set in the `toolkit.properties` file:
 
--   `<database_id>` — The name of the target database.  Set this to the name of the database in your cluster that you want to use as your migration target database.  The name of the default database for your cluster is shown in the **Dbname** field on the **Connect** tab in the BigAnimal portal.  Often a separate database is created for use as the migration target. 
+-   `TARGET_DB_USER` — Specifies the name of a privileged database user. You can copy it from the **User** field on the **Connect** tab in the Cloud Service portal.
 
--   `TARGET_DB_USER` — Specifies the name of a privileged database user. You can copy it from the **User** field on the **Connect** tab in the BigAnimal portal.
+-   `TARGET_DB_PASSWORD` — Contains the password of the specified user. Provide the password without any surrounding quotes.
 
--   `TARGET_DB_PASSWORD` — Contains the password of the specified user. 
+Ensure this value is set in Cloud Service:
```

Comment:
The `sslmode` parameter is part of the connection URL for cloud service connections. I think this should be moved up to the text that describes the connection URL settings.

Reply:
Good catch, I thought it was a Console configuration. Updated ✅